Content.Client/VendingMachines/UI/VendingMachineMenu.xaml.cs

```diff
@@ -45,7 +45,7 @@ public VendingMachineMenu()
         /// Populates the list of available items on the vending machine interface
         /// and sets icons based on their prototypes
         /// </summary>
-        public void Populate(List<VendingMachineInventoryEntry> inventory, float priceModifier, string? filter = null)
+        public void Populate(List<VendingMachineInventoryEntry> inventory, string? filter = null, float priceModifier = 10)
```

Comment:
nullable fields should be listed after non-nullables to allow truncating the function calls.  Also, you're bulldozing the price modifier mod by hard-coding it to be permanently 10, discarding any input

Reply:
Didnt notice its not taking the floor from other mod changes, will redo it